src/tests/test_suggestions_service.py

```
@@ -0,0 +1,104 @@
+    job_table = Job.__table__.to_metadata(scratch_metadata)
+    for col in job_table.columns:
+        if isinstance(col.type, JSONB):
+            col.type = JSON()
+        elif isinstance(col.type, ARRAY):
+            col.type = Text()
+        elif isinstance(col.type, UUID):
+            col.type = VARCHAR(36)
+
+    # Register PostgreSQL's now() function for SQLite
+    @event.listens_for(engine.sync_engine, "connect")
+    def register_now(dbapi_conn, connection_record):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `register_now`

Add return type annotation: `None`

(ANN202)